### ISSUE TITLE
Intermittent travis issues win32/win64

### ISSUE THREAD

**Issue by laanwj** [bitcoin/bitcoin#7924]
This crops up a lot lately:
- https://travis-ci.org/bitcoin/bitcoin/jobs/124945684
- https://travis-ci.org/bitcoin/bitcoin/jobs/124945686

Either timeouts, or
```
E: Unable to locate package wine1.7
E: Couldn't find any package by regex 'wine1.7'
The command "sudo apt-get install --no-install-recommends --no-upgrade -qq nsis gcc-mingw-w64-x86-64 g++-mingw-w64-x86-64 binutils-mingw-w64-x86-64 mingw-w64-dev wine1.7 bc" failed. Retrying, 2 of 3.
```

**Comment by theuni** [comment]
The trusty build won't rely on this repo anymore, so this can be closed with #7920

**Comment by MarcoFalke** [comment]
Closing per previous comment
